x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/types.ts

```diff
@@ -174,3 +180,42 @@ export interface SelectedIndex {
   indexName: string;
   pattern: string;
 }
+
+export interface DataQualityIndexCheckedParams {
```

Comment:
Consider something like

```ts
export type DataQualityIndexCheckedParams = DataQualityCheckAllCompletedParams & {
  errorCount?: number;
  ilmPhase?: string;
  indexId: string;
  unallowedMappingFields?: string[];
  unallowedValueFields?: string[];
};
```

because it appears that `DataQualityIndexCheckedParams` is a superset of `DataQualityCheckAllCompletedParams`